server/controllers/finance/account.js

```diff
@@ -0,0 +1,80 @@
+var db = require('../../lib/db');
+
+function create (req, res, next){
+  var record = req.body;
+  var create_account_query = 'INSERT INTO account SET ?';
+  var transaction = db.transaction();
+
+  transaction
+  .addQuery(create_account_query, [record]);
+
+  transaction.execute()
+  .then(function (results){
+    var confirmation = results;
+    res.status(201).json(confirmation);
```

Comment:
What will this return?  Why do you need to make a local variable `confirmation`?